app/controllers/report_controller/reports/editor.rb

```diff
@@ -628,6 +628,10 @@ def gfv_performance
     end
   end
 
+  def chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
@zeari  nice to know,  ok so after this will be merged https://github.com/ManageIQ/manageiq/pull/8818 
there can be used `Chargeback.subclasses` because we are loading there subclasses